car_data.rb

```diff
@@ -0,0 +1,92 @@
+class Car
+
+  attr_reader :name, :gas_mileage, :n_cylinders, :horsepower, :weight, :quarter_mile_time, :salesman_name
```

Comment:
I'd leave salesman_name off of the attr_reader list, and rather than creating it as an instance variable during initialization, just have a method called salesman_name which constructs the value. If you're worried about the cost of that computation being repeated, you can set an instance var inside the method like so:
@salesman_name ||= #computation goes here